### DISCUSSION TITLE
Support for Intel GPUs

### DISCUSSION THREAD

**Discussion by psmgeelen** [intel/scikit-learn-intelex#1048] [Q&A]
#EDIT: Added 'Arc' specification as it should relate to the new discrete Intel GPUs
Hi there, 
I am excited about the Intel Arc GPUs and noticed that they use the same `oneAPI` that is fundamental to this library. Will the Intel Arc GPUs be supported in this library? They might have a compelling case, as 3rd Gen Intel Xeons can apparently already outperform a nVidia A100 GPU [source](https://community.intel.com/t5/Blogs/Tech-Innovation/Artificial-Intelligence-AI/New-3rd-Gen-Intel-Xeon-Scalable-Processors-Demonstrate-Machine/post/1335746)

Thanks for considering my question
Regards

**Comment by Alexsandruss** [comment]
Hi @psmgeelen, scikit-learn-intelex supports Intel(R) GPUs with some [requirements](https://intel.github.io/scikit-learn-intelex/oneapi-gpu.html).

**Reply by psmgeelen** [reply]
Hi there, I noticed in the documentation that there isn't any mention for the new discrete (non-integrated) Arc GPUs that are coming out, (source)[https://www.intel.com/content/www/us/en/products/details/discrete-gpus/arc.html]. Is there any news on that?

**Reply by napetrov** [reply]
they are also supported and documentation would be updated
